frontend/packages/console-shared/src/components/catalog/catalog-view/CatalogTypeSelector.tsx

```diff
@@ -49,7 +50,7 @@ const CatalogTypeSelector: React.FC<CatalogTypeSelectorProps> = ({
             search: `?${queryParams.toString()}`,
           };
 
-          return typeCount > 0 ? (
+          return typeCount > 0 && !disabledSubCatalogs?.includes(value) ? (
```

Comment:
Instead of keeping the `catalogTypes` prop with all items from `CatalogController` to `CatalogView` and hiding the values here, could we filter these already in `CatalogController`?

The code there looks like this and should be easy to filter, or?

```tsx
  const catalogTypes: CatalogType[] = React.useMemo(() => {
    const types = catalogExtensions.map((extension) => ({
      label: extension.properties.title,
      value: extension.properties.type,
      description: extension.properties.typeDescription,
    }));

    return _.sortBy(types, ({ label }) => label.toLowerCase());
  }, [catalogExtensions]);
```

(untested)